doc/configuration.rst

```diff
@@ -411,6 +411,46 @@ hive resources and 1 mysql resource:
 Note that it was not necessary to specify the 1 for mysql here, but it
 is good practice to do so when you have a fixed set of resources.
 
+.. _retcode-config:
+
+[retcode]
+----------
+
+Configure return codes for the luigi binary. In the case of multiple return
+codes that could apply, for example a failing task and missing data, the
+*numerically greatest* return code is returned.
+
+We recommend that you copy this set of exit codes to your ``luigi.cfg`` file:
+
+::
+
+  [retcode]
+  # The following return codes are the recommended exit codes for luigi
+  # They are in increasing level of severity (for most applications)
+  already_running: 1
+  missing_data: 2
+  task_failed: 3
+  unhandled_exception: 4
```

Comment:
Prefer to spread them out, 10 20 30 40 like BASIC line numbers or some other scheme? to give us more leeway in introducing codes for nuances of any of the conditions in the future, without breaking the increasing level of severity (approximate) invariant.